db-constraints/src/main/java/org/evomaster/constraint/AndConstraint.java

```diff
@@ -0,0 +1,14 @@
+package org.evomaster.constraint;
+
```

Comment:
when we have a module, the package name should be matching somehow the name of the of module.
Here, the package is `constraint` whereas the module is `db-constraints`. I think it would be better if we rename the package with something like `org.evomaster.dbconstraint` (should not have `-` in the name of a package)

Reply:
Thanks. I renamed the module to "dbconstraint" and the package to org.evomaster.dbconstraint to satisfy the naming convention.